### ISSUE TITLE
Command Line

### ISSUE THREAD

**Issue by he3als** [Nilsen84/lunar-client-qt#51]
Adding the ability to launch Lunar Client from LC-QT with the command line would be great (with no GUI).

**Comment by ceilingfans** [comment]
Why is there a need for this?

**Comment by he3als** [comment]
You would be able to easily launch Lunar Client with LC-Qt with the Run box or a shortcut, making it so you can launch Lunar Client right away, instead of going through a GUI.

**Comment by ceilingfans** [comment]
This is a valid point however the goal for lcqt is to be lightweight and adding a CLI will only add bloat. Having a CLI doesn't bring that many advantages since only a handful of people would actually use it.

**Comment by he3als** [comment]
How would something that never would get in your way be bloat for a normal user? You could argue that a whole GUI is bloat to launch Lunar when you can have no GUI.

**Comment by ceilingfans** [comment]
You're right in saying it's not bloat for I used the wrong term here. My main point is that very few people would actually use the CLI while everyone can use the GUI so it seems unnecessary to me. However it's up to nils to decide whether it is useful enough to implement.
